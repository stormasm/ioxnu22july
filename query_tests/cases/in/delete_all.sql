-- Demonstrate soft deleted rows will not be return to queries
-- IOX_SETUP: OneDeleteSimpleExprOneChunkDeleteAll

-- select *
SELECT * from cpu;

-- select one specific column
SELECT time from cpu;

-- select aggregate of every column inlcuding star
SELECT count(*), count(bar), count(time) from cpu;

-- select aggregate of every column
SELECT min(bar), max(bar), min(time), max(time) from cpu;

-- select aggregate of one column
SELECT max(bar) from cpu;