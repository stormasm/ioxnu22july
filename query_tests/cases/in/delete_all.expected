-- Test Setup: OneDeleteSimpleExprOneChunkDeleteAll
-- SQL: SELECT * from cpu;
++
++
-- SQL: SELECT time from cpu;
++
++
-- SQL: SELECT count(*), count(bar), count(time) from cpu;
+-----------------+----------------+-----------------+
| COUNT(UInt8(1)) | COUNT(cpu.bar) | COUNT(cpu.time) |
+-----------------+----------------+-----------------+
| 0               | 0              | 0               |
+-----------------+----------------+-----------------+
-- SQL: SELECT min(bar), max(bar), min(time), max(time) from cpu;
+--------------+--------------+---------------+---------------+
| MIN(cpu.bar) | MAX(cpu.bar) | MIN(cpu.time) | MAX(cpu.time) |
+--------------+--------------+---------------+---------------+
|              |              |               |               |
+--------------+--------------+---------------+---------------+
-- SQL: SELECT max(bar) from cpu;
+--------------+
| MAX(cpu.bar) |
+--------------+
|              |
+--------------+
